package.json

```diff
@@ -42,19 +42,19 @@
     "eslint-plugin-node": "^8.0.1",
     "eslint-plugin-prettier": "^3.0.1",
     "eslint-plugin-promise": "^4.0.1",
-    "fastify": "^2.0.0",
-    "hemera-plugin": "^2.0.0",
+    "fastify": "^3.2.0",
+    "hemera-plugin": "^2.0.2",
     "hemera-testsuite": "^4.1.0",
-    "light-my-request": "^3.2.0",
+    "light-my-request": "^4.0.1",
     "np": "^4.0.2",
     "pre-commit": "^1.2.2",
     "prettier": "^1.16.4",
     "tap-spec": "^5.0.0"
   },
   "dependencies": {
-    "fastify-plugin": "^1.5.0",
-    "nats": "1.2.2",
-    "nats-hemera": "^7.0.0",
-    "tap": "^12.5.3"
+    "fastify-plugin": "^2.2.0",
+    "nats": "1.4.12",
+    "nats-hemera": "^7.0.2",
+    "tap": "^14.10.8"
```

Comment:
This is a dev dep.

Reply:
Sure, reason for upgrade is that tests are using those too... at least fastify-plugin needs an upgrade to support fastify 3 - new tap comes with built in spec reporter